src/index.js

```diff
@@ -621,22 +622,23 @@ class Manager {
       return this._spawn(binPath, sanityCheck.args);
     })
     .then((output) => {
-      const haystack = output.stdout + output.stderr;
+      this._logger.debug(`Sanity check output: ${output.stdout + output.stderr}`);
 
-      this._logger.debug(`Sanity check output: ${haystack}`);
-
-      const needles = sanityCheck.output || [];
+      try {
+        var version = output.stdout.match(/\d+\.\d+\.\d+/)[0];
+      } catch (err) {
+        throw new Error(`Unable to find semantic version >= "${client.version}" in "${binPath} ${sanityCheck.args}"'s' output`);
+      }
 
-      for (let needle of needles) {
-        if (0 > haystack.indexOf(needle)) {
-          throw new Error(`Unable to find "${needle}" in ${client.id} output`);
-        }
+      if (version && semver.lt(version, client.version)) {
```

Comment:
You are right, that would be better. 

Though, I can only think of using `stdout.match(/\d+\.\d+\.\d+/)[0]` for the version string hoping that every `<node> [--]version` will have its version as the first semantic version string in the output. 

Eventually the haystack-method is fine, but then a method to retrieve the system node's version via a regex in the `clientBinaries.json` might bring a bit more flexibility.

Alternatively we could omit the version checking and supply an optional boolean `--uselocalnode` quite similar to `--gethpath`.

What do you think?